### PR TITLE
[primTorch] asinh, atanh

### PR DESCRIPTION
**Pull request by khushi-411**
Adds asinh and atanh refs and prims. These are prims because both are C++ standard library calls.

### CODE REVIEW

**Review by mruberry — APPROVED** [review]
Cool! Thanks @khushi-411 

cc @kevinstephano

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -20051,10 +20051,20 @@ def __init__(
...
+        supports_nvfuser=False,
+    ),
     ElementwiseUnaryPythonRefInfo(
         "_refs.atan",
         torch_opinfo_name="atan",
     ),
+    ElementwiseUnaryPythonRefInfo(
+        "_refs.atanh",
+        torch_opinfo_name="atanh",
+        supports_nvfuser=False,
```
[the remark this answers is not in the document]
Comment (khushi-411) [reply torch/testing/_internal/common_methods_invocations.py:20066]: Hi, @IvanYashchuk! Yes, you are right. Thanks for pointing that out :) I forgot to remove it while I was resolving failure in the PR. I'll send another PR removing it.

**Pull request closed without merging**